beacon/handler.go

```diff
@@ -71,10 +71,8 @@ func NewHandler(c net.ProtocolClient, s Store, conf *Config, l log.Logger) (*Han
 	if node == nil {
 		return nil, errors.New("beacon: keypair not included in the given group")
 	}
-	idx := node.Index
 	addr := conf.Public.Address()
-	// XXX change logging because of resharing
-	logger := l.With("index", idx)
```

Comment:
is this not a useful flag?

Reply:
It's useful for debugging on local tests but not useful to debug on a distributed setting. Main problem is that it changes during resharing and the log showed the previous one so i prefer to remove it than showing something outdated - it distracted me for some time :D